src/reanimated2/jestUtils.ts

```diff
@@ -6,34 +6,67 @@ let config = {
   fps: 60,
 };
 
-const checkValidation = (received) => {
-  let message = [];
-  let isValid = true;
-  if (!received.props.style) {
-    isValid = false;
-    message.push("Component not contains 'style' property");
-  }
-  if (!received.props.animatedStyle?.value) {
-    isValid = false;
-    message.push("Component not contains 'animatedStyle' property");
-  }
-  message = message.join('\n');
+const isAnimatedStyle = (style) => {
+  return !!style.animatedStyle;
+};
 
-  return { isValid, message };
+const getAnimatedStyleFromObject = (style) => {
+  return style.animatedStyle.current.value;
 };
 
 const getCurrentStyle = (received) => {
-  return {
-    ...received.props.style,
-    ...received.props.animatedStyle.value,
-  };
+  const styleObject = received.props.style;
+  let currentStyle = {};
+  if (Array.isArray(styleObject)) {
+    received.props.style.forEach((style) => {
+      if (isAnimatedStyle(style)) {
+        currentStyle = Object.assign(
+          {},
+          currentStyle,
+          getAnimatedStyleFromObject(style)
+        );
```

Comment:
Why we use `Object.assign` here? This is Jest so we should be able to use spread too.